<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Seja Bem-Vindo!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Efetue Um novo Cadastro:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="CpNome">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Nome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NOME:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CPF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CPF:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CpCpf">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CpRg">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CpEnd">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CpSetor">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_END">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>END:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_RG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RG:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SETOR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SETOR:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_LOGUIN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LOGIN:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CpLoguin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_SENHA">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SENHA:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CpSenha">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CpSalario">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Salario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SALÁRIO:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>821</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tabela">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>701</width>
       <height>181</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>NOME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>END</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CPF</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RG</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SETOR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>USUÁRIO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SALARIO</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="BtnRemover">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Remover</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Ordenar">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>200</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordenar por nome</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>200</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordenar por Salário</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Listagem de Funcionários:</string>
    </property>
   </widget>
   <widget class="QLabel" name="banco">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnRegistrar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Maior Salário:</string>
     </property>
    </widget>
    <widget class="QLabel" name="MostrarSalario">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Média de Salário:</string>
     </property>
    </widget>
    <widget class="QLabel" name="MostrarMedia">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Menor Salário:</string>
     </property>
    </widget>
    <widget class="QLabel" name="menorSal">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dados Estatísticos:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionCarregar"/>
    <addaction name="actionSalvar_3"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionRegistrar_Sa_da">
   <property name="text">
    <string>Registrar Saída</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionSalvar_2">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="Salvar" stdset="0">
    <string/>
   </property>
  </action>
  <action name="actionAbrir_2">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSalvar_3">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
